pandas/tests/io/msgpack/test_case.py

```diff
@@ -7,9 +7,7 @@ def check(length, obj):
     v = packb(obj)
     assert (
         len(v) == length
-    ), "{obj!r} length should be {length!r} but get {got:!r}".format(
-        obj=obj, length=length, got=len(v)
-    )
+    ), f"{repr(obj)} length should be {repr(length)} but got {repr(len(v))}"
```

Comment:
do the reprs matter here?

Reply:
same question in a bunch of places i guess

Reply:
> do the reprs matter here?

No, well... not REALLY important, I just convert the old syntax to the new syntax.
I usually don't think about what I'm changing after a few, I get to "auto" mode.

The goal for now (for me) is to eliminate all the occurrences of ```!r```, so we can add a test case that will prevent the use of ```!r``` anywhere in the code.

Reply:
> and after a few, I get to "auto" mode

I totally understand that phenomenon.  The failure mode to worry about here is these not getting any attention for 2 years and the next person can't figure out why an apparently-unneeded `repr` is there, but assumes the last author had a good reason.

Reply:
@jbrockmendel You're way more experience than I am, your call.

Reply:
can you change the repr(length) to just length here